src/content/en/fundamentals/media/mse/basics.md

```diff
@@ -91,6 +91,11 @@ Here, in no particular order, are a few things I won't cover.
    `<audio>` and `<video>` elements.
 +  Error handling for the sake of simplicity and clarity.
 
+Warning: Handling error events, API exceptions, and proactively checking
+`HTMLMediaElement readyState` and `MediaSource readyState` values before making
+calls on their APIs are mandatory in a production usage of the APIs as these
+values can change before associated events are delivered.
+
```

Comment:
How straight forward is it to implement this? I'm wondering if this topic needs a minor section with a code example.